data/human/culture conversations.txt

```diff
@@ -1254,3 +1254,375 @@ mission "Little Vienna in New Austria"
+				"syndicate speed" = "roll: 10" + "roll: 10" + "roll: 10" + ( ( 10 - 4 + "race length" ) / 10 ) * ( "roll: 10" ) + ( ( 10 - 5 + "race length" ) / 10 ) * "roll: 10"
+				"delta speed" = "roll: 8" + "roll: 8" + "roll: 8" + ( ( 10 - 4 + "race length" ) / 10 ) * ( "roll: 8" ) + ( ( 10 - 5 + "race length" ) / 10 ) * "roll: 8"
+				"deep speed" = "roll: 6" + "roll: 6" + "roll: 6" + ( ( 10 - 4 + "race length" ) / 10 ) * ( "roll: 6" ) + ( ( 10 - 5 + "race length" ) / 10 ) * "roll: 6"
+				"syndicate laps" = "syndicate speed" + "syndicate boost"
+				"delta laps" = "delta speed" + "delta boost"
+				"deep laps" = "deep speed" + "deep boost"
+			`	You board <ship> for the quick trip to the <racename>. Out the window, ${sunracer city to wilderness description}. Eventually, you park your ship outside the <racename>, and walk up to the racetrack's entrance.`
+			label decisioning
+			`	The screen above flashes race details:`
+			`		<racename>: &[race length] Hour Race`
+			``
+			`		Today's Racers:`
+			`		Megaparsec Racing Technologies: <megaparsec name>`
+			`		Syndicate Performance Racing: <syndicate name>`
+			`		Delta V Grand Prix: <delta name>`
+			`		Deep Sky Dynamics: <deep name>`
+			``
+			choice
+				`	(Watch the race.)`
+					goto race
+				`	(Ask someone for an overview of hover racing.)`
+					to display
+						has "hovercraft racing: talked to fan"
+				`	(Use your ship to scan the racers.)`
+					to display
+						or
+							"flagship attribute: outfit scan power" > 0
+							"flagship attribute: tactical scan power" > 0
+					goto scanner
+			action
+				set "hovercraft racing: talked to fan"
+			`	Near the <racename>'s entrance, you spot a friendly figure ${sunracer fan tshirt}:`
+			`	"Hovercraft racing pits four ships against the clock - each trying to log as many laps as possible. The race is between three and five hours long and is split into two phases: an initial afterburn for a burst of speed, followed by laps completed in the remaining time.`
+			`	"Each racer has a roughly equal chance of victory, though Megaparsec Racing Technologies tends to win a bit more often overall. Megaparsec also excels in longer races thanks to their efficient ion engines that can go anywhere from zero to twelve laps an hour, but completely lack any initial speed burst. Deep racers shine on shorter stints by using high-powered atomics for a powerful start that can push them to complete between six and eighteen laps almost right out of the gate, but in longer races their lack of efficiency means they go only half as fast as Megaparsec, anywhere from zero to six laps an hour. The other two teams fall somewhere in-between.`
+			`	"Even though each team is roughly equal, in a given race, a savvy watcher can better predict a winner by knowing which teams perform better on shorter or longer race lengths, or a less sporting one with a powerful scanner might even be able to tell a hovercraft's exact afterburn level before the race even begins.`
```

Comment:
This sounds like technobabble. What kind of afterburner do hovercraft use? Does "afterburn level" refer to something like the hyperspace fuel they have loaded?

Reply:
Meaning "afterburn level" to imply "level of afterburner fuel," but likely better to just explicitly say that then:
```suggestion
			`	"Even though each team is roughly equal, in a given race, a savvy watcher can better predict a winner by knowing which teams perform better on shorter or longer race lengths, or a less sporting one with a powerful scanner might even be able to tell a hovercraft's exact level of afterburner fuel before the race even begins.`
```